[VI] Add `Community/How you can help` (#11439)
ppy/osu-wiki: wiki/Community/How_you_can_help!/vi.md

Thật may là việc giúp giữ mọi thứ an toàn ở đây khá là dễ dàng. Sau đây chỉ là một trong số rất nhiều cách bạn có thể giúp:

Theo dõi các kênh yêu thích của bạn ở [phần trò chuyện (chat) trong game](/wiki/Client/Interface/Chat_console) hoặc tham gia vào các diễn đàn.

Nếu bạn thấy điều gì đó xảy ra mà bạn cho là không phù hợp hoặc gây phiền phức theo bất kỳ cách nào, hãy sử dụng [lệnh `!report`](/wiki/Reporting_bad_behaviour) trong phần trò chuyện để thông báo cho kiểm duyệt viên về điều đó.

## Giúp đỡ các thành viên khác trong cộng đồng

Nếu bạn thấy ai đó đang gặp vấn đề với game, hãy giúp họ một tay! Cho dù đó là vấn đề về phần cứng, câu hỏi về game hoặc beatmap, nếu bạn có thể trả lời được thì bạn có thể giúp họ giải quyết vấn đề.

Sau đây là các gợi ý để bạn có thể bắt đầu giúp đỡ người khác trên diễn đàn:

- Chọn một diễn đàn thu hút sự quan tâm của bạn, ví dụ:
 - [Help (Diễn đàn Trợ giúp)](https://osu.ppy.sh/community/forums/5)
 - [Mapping Discussion (Diễn đàn Thảo luận Beatmap)](https://osu.ppy.sh/community/forums/56)
 - [Storyboarding](https://osu.ppy.sh/community/forums/20)

>  - [skinning](https://osu.ppy.sh/community/forums/15)

Reviewer: ```suggestion
 - [Skinning](https://osu.ppy.sh/community/forums/15)
```

 - [Gameplay & Rankings](https://osu.ppy.sh/community/forums/13)
- Đọc qua các bài đăng được ghim ở diễn đàn để tìm hiểu các quy tắc cụ thể dành riêng cho từng diễn đàn nhỏ và những thông tin hữu ích ở trong bài đăng đó.

> -Sử dụng nút `Mark forum as read` ở trên cùng.

Reviewer: ```suggestion
- Sử dụng nút `Mark forum as read` ở trên cùng.
```

> - Theo dõi tất cả các chủ đề mới, đọc từng chủ đề và trợ giúp khi có thể  — bạn có thể sử dụng biểu tượng chưa đọc để chắc chắn rằng bạn bắt kịp với mọi thông tin.

Reviewer: ```suggestion
- Theo dõi tất cả các chủ đề mới, đọc từng chủ đề và trợ giúp khi có thể — bạn có thể sử dụng biểu tượng chưa đọc để chắc chắn rằng bạn bắt kịp với mọi thông tin.
```

 - Tham gia [osu!discord](https://discord.com/invite/ppy) và chỉ cần theo dõi cập nhật phát triển osu! và các dự án là một cách tốt để có thông tin về các dự án trong tương lai.
- Tham gia [dự án beatmap](https://osu.ppy.sh/community/forums/53) để tìm hiểu việc hợp tác với người khác thú vị như thế nào!
- Bắt đầu dự án của riêng bạn hoặc tham gia dự án của người khác! Cho dù đó là về những chú pippi nhảy múa hay đơn giản là làm thứ gì đó thú vị cho osu!, mọi thứ có thể được chấp nhận!
 - Kiểm tra [danh sách các dự án phổ biến](/wiki/Community/Projects).

## Đóng góp cho tương lai của osu!

Bắt đầu vào năm 2017, dự án [osu!(lazer)](/wiki/Client/Release_stream/Lazer) là trọng tâm hiện tại của [osu!team](/wiki/People/osu!_team) và cùng với wedsite mới, đó là những gì bạn thấy ngày hôm nay! Nếu bạn muốn làm cho những dự án này trở nên tốt hơn nữa và đến gần hơn với tương lai của osu!, thì bạn có thể giúp một số việc dưới nhưng đây chỉ là một vài ví dụ:

- Giúp đỡ dự án [osu!(lazer)](https://github.com/ppy/osu). Có rất nhiều thứ cần triển khai trên phiên bản mới của osu!, và bạn có thể giúp biến điều đó thành hiện thực!

> - Làm cho [wedsite](https://github.com/ppy/osu-web) ngày càng tốt hơn.

Reviewer: ```suggestion
- Làm cho [trang web](https://github.com/ppy/osu-web) ngày càng tốt hơn.
```

## Tránh tiêu cực! Làm cho mọi người vui vẻ!

> Có một câu nói cổ rằng :*“bạn nên trở thành một phần của giải pháp chứ không phải của vấn đề”*. Chỉ cần làm theo câu nói cổ này, bạn sẽ giúp được osu! bằng cách làm cho cộng đồng trở thành một nơi tươi sáng hơn. Mọi người sẽ luôn phàn nàn, nhưng vẫn có rất nhiều cách bạn có thể giúp cải thiện tâm trạng trong cộng đồng:

Reviewer: ```suggestion
Người xưa có câu "hãy là một phần của giải pháp, đừng là một phần của vấn đề". Chỉ cần làm theo câu nói này, bạn sẽ giúp được osu! bằng cách làm cho cộng đồng trở thành một nơi tươi sáng hơn. Mọi người sẽ luôn phàn nàn, nhưng vẫn có rất nhiều việc bạn có thể làm để giúp cải thiện tâm trạng của mọi người:
```

- Hãy thân thiện với nhau; cuộc sống quá ngắn để tranh luận.

> - [Báo cáo những kẻ gian lận khi bạn nhìn thấy chúng](/wiki/Reporting_bad_behaviour/Handling_foul_play). Vui lòng không phàn nàn về chúng thông qua các phương tiện khác vì có một cách đơn giản là`!report <username> seems to be cheating`có thể giúp chúng bay màu trong vòng vài phút đến vài giờ.

Reviewer: ```suggestion
- [Báo cáo những kẻ gian lận khi bạn nhìn thấy chúng](/wiki/Reporting_bad_behaviour/Handling_foul_play). Vui lòng không phàn nàn về những người này thông qua các phương tiện khác vì chỉ cần một câu đơn giản là `!report <username> seems to be cheating` là đủ để vấn đề của bạn đi khuất mắt chỉ trong vài phút đến vài giờ.
```

- Hãy cố gắng hòa giải các tình huống tiêu cực và biến chúng thành kết quả tích cực.

> **Mặc dù bài viết này không thể đề cập đến việc bạn có thể giúp được bao nhiêu trong cộng đồng osu!, chỉ cần chơi game cũng có thể giúp osu! thậm chí còn tuyệt vời hơn nữa!**( đừng nghiện game quá, lol)

Reviewer: ```suggestion
**Mặc dù bài viết này không thể đề cập tất cả các cách để bạn có thể đóng góp cho cộng đồng của osu!, nhưng chỉ cần chơi game thôi cũng có thể giúp osu! thậm chí còn tuyệt vời hơn nữa!**
```
